docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
   agd:
     # image publication is WIP
     # cf. https://github.com/Agoric/agoric-3-proposals/issues/6
```

Comment:
_question_

Is this issue still relevant?

Reply:
good catch. the comment is stale